src/pages/Dashboard.vue

```diff
@@ -105,12 +105,38 @@
               <p class="h-4 w-1/2 place-self-center bg-yellow-900 rounded-md mt-1"></p>
             </div>
             <div v-else>
+              <!-- value -->
               <p :class="cardMetricValueClass">
                 {{ completionRate }}
               </p>
-              <p :class="cardMetricTitleClass">
-                {{ $t("dashboard.summary.completion_rate") }}
-              </p>
+              <div :class="cardMetricTitleClass" class="flex justify-center">
+                <div
+                  class="flex flex-col md:flex-row space-y-2 md:space-y-0 md:space-x-2 items-center"
+                >
+                  <!-- title -->
+                  <p>
+                    {{ $t("dashboard.summary.completion_rate.title") }}
+                  </p>
+                  <div class="flex relative">
+                    <!-- info icon -->
+                    <inline-svg
+                      :src="require('@/assets/images/info.svg')"
+                      class="h-4 w-4 text-yellow-900 fill-current hover:cursor-pointer"
+                      @mouseover="showCompletedHelpText = true"
+                      @mouseout="showCompletedHelpText = false"
+                      @touchmove="showCompletedHelpText = true"
+                      @touchend="showCompletedHelpText = false"
+                    ></inline-svg>
+                    <!-- info text -->
+                    <p
+                      class="bg-black text-white rounded-md p-2 absolute ml-4 w-28"
```

Comment:
<img width="253" alt="Screenshot 2021-05-19 at 7 46 47 PM" src="https://user-images.githubusercontent.com/20169829/118828513-f5bb3480-b8da-11eb-9c97-b81a90de5f6c.png">
Some css issue?

Reply:
<img width="440" alt="Screenshot 2021-05-19 at 7 47 31 PM" src="https://user-images.githubusercontent.com/20169829/118828632-0ff51280-b8db-11eb-94b5-06bf511bdfbc.png">
In 320px mode, the info text is being cut off on the right side